### PR TITLE
Eb sprint 2 fetch pantry

### PR DESCRIPTION
**Pull request by ericberglund117**
## Description

- Added modifyPantry HTML. This includes modifyPantry button as well
- Updated pantry html styles in styles.scss
- Create modifyPanty functionality in  User class to update user's pantry 
- Created fetch post request of modified pantry data 

## Motivation and Context

- This now takes the modified pantry data and posts it to the server via a fetch request.
- We first updated the User class and then the fetch request 

## How Has This Been Tested?

-  Tested through the browser 
- fetch promises were resolved, no error caught

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -407,19 +414,100 @@ function findRecipesWithCheckedIngredients(selected) {
...
+    amounts.forEach(amount => {
+      if (amount.value && amount.value !== 0) {
+        let ingredID = amount.parentNode.parentNode.id;
+        let ingredMod = amount.value;
+        console.log('id', ingredID, 'value', ingredMod)
+        if(ingredMod > 0 || ingredMod < 0){
+          currentUser.updateCurrentUserPantry(ingredID, ingredMod);
+          updatePantryIngredients(ingredID, ingredMod)
+        }
+      }
```
Comment (Kristiannmiller) [inline src/scripts.js:489]: Let's do a display: non style change on this html element once this button has been pressed as well.

**Comment by Kristiannmiller** [comment]
This pull request is magic! Good work!! Let's move on to DOM Manipulation with this next branch

File: src/scripts.js [hunk 2]
```diff
@@ -22,6 +22,8 @@ let searchForm = document.querySelector("#search");
 let searchInput = document.querySelector("#search-input");
 let showPantryRecipes = document.querySelector(".show-pantry-recipes-btn");
 let tagList = document.querySelector(".tag-list");
+let modifyPantryBtn = document.querySelector(".modify-pantry-btn")
+let searchIngBtn = document.querySelector(".search-ingredients-btn")
```
Comment (chadrick-d-dev) [inline src/scripts.js:26]: Is Ing descriptive enough?

File: src/scripts.js [hunk 3]
```diff
@@ -407,19 +414,100 @@ function findRecipesWithCheckedIngredients(selected) {
...
+  };
+
+  function submitPantryChanges(event) {
+  if (event.target && event.target.id === 'save-changes-btn') {
+    let amounts = Array.from(document.querySelectorAll('.amount'));
+    amounts.forEach(amount => {
+      if (amount.value && amount.value !== 0) {
+        let ingredID = amount.parentNode.parentNode.id;
+        let ingredMod = amount.value;
+        console.log('id', ingredID, 'value', ingredMod)
```
Comment (chadrick-d-dev) [inline src/scripts.js:484]: Let's look at these parameters, are we practicing empathy using partial words?

File: src/scripts.js (hunk 2 above)
Comment (Kristiannmiller) [reply src/scripts.js:26]: Agreed! Let's tackle this in our big refactor near the end of the project

**Review by chadrick-d-dev — APPROVED** [review]
Check out my comments. Great job everyone!

**Pull request merged**